reviews/controller.py

```diff
@@ -28,15 +29,37 @@ def retrieve_pull_requests(self, org: str, repository: str) -> Union[Table, None
 
     def update_pull_requests(self, org: str, repository: str) -> List[PullRequest]:
         """Updates repository models."""
-        pull_requests = self.client.get_pull_requests(org=org, repo=repository)
 
+        def _get_reviews(pull_request):
+            """Inner function to retrieve reviews for a pull request"""
+            reviews = sorted(
+                pull_request.get_reviews(), key=lambda r: r.submitted_at, reverse=True
+            )
+            res, seen = {}, []
+            for review in reviews:
+                if review.user.login in seen or review.state == "COMMENTED":
+                    continue
+                res[review.user.login] = review.state
+                seen.append(review.user.login)
+            return res
+
+        pull_requests = self.client.get_pull_requests(org=org, repo=repository)
         return [
             PullRequest(
                 number=pull_request.number,
                 title=pull_request.title,
                 created_at=pull_request.created_at,
                 updated_at=pull_request.updated_at,
-                approved=any(pull_request.get_reviews()),  # NOQA: R1721
+                approved=_get_reviews(pull_request).get(
```

Comment:
named parameters are prefered over positional - it makes extension easier in the future. 

```suggestion
                approved=_get_reviews(pull_request=pull_request).get(
```